tests/Umbraco.Tests.Integration/ManagementApi/Services/Trees/StyleSheetTreeServiceTests.cs

```diff
@@ -1,27 +1,57 @@
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
 using NUnit.Framework;
 using Umbraco.Cms.Api.Management.Services.FileSystem;
 using Umbraco.Cms.Api.Management.ViewModels.Tree;
+using Umbraco.Cms.Core.Configuration.Models;
 using Umbraco.Cms.Core.IO;
+using Umbraco.Cms.Tests.Common.TestHelpers;
 
 namespace Umbraco.Cms.Tests.Integration.ManagementApi.Services.Trees;
 
 public class StyleSheetTreeServiceTests : FileSystemTreeServiceTestsBase
 {
     protected override string FileSystemPath => GlobalSettings.UmbracoCssPath;
 
+    protected FileSystems FileSystems { get; private set; }
+
+    protected IFileSystem TestFileSystem { get; private set; }
+
     protected override IFileSystem? GetStylesheetsFileSystem() => TestFileSystem;
 
+
+    [SetUp]
+    public override void SetUpFileSystem()
+    {
+        TestFileSystem = new PhysicalFileSystem(IOHelper, HostingEnvironment, LoggerFactory.CreateLogger<PhysicalFileSystem>(), HostingEnvironment.MapPathWebRoot(FileSystemPath), HostingEnvironment.ToAbsolute(FileSystemPath));
+
+        FileSystems = FileSystemsCreator.CreateTestFileSystems(
+            LoggerFactory,
+            IOHelper,
+            GetRequiredService<IOptions<GlobalSettings>>(),
+            HostingEnvironment,
+            GetPartialViewsFileSystem(),
+            GetStylesheetsFileSystem(),
+            GetScriptsFileSystem(),
+            null);
+        for (int i = 0; i < 10; i++)
+        {
+            using var stream = CreateStream(Path.Join("tests"));
+            TestFileSystem.AddFile($"file{i}.css", stream);
+        }
+    }
```

Comment:
Maybe you could avoid copying all of this from the base class by instead adding an abstract property for the file extension to the base class.  That'll be used in the base class for this setup, and you can override it in derived classes to set to be `.css` or whatever you need.